_data/flights.toml

```diff
@@ -0,0 +1,57 @@
+[[specs]]
+# This spec won't be part of the final PR.
+# Using it to demonstrate the deviation from [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601)
+range     = [2001-01-01, 2001-03-31]
+n_rows    = 1_000
+suffix    = ".csv"
+dt_format = "ISO"
```

Comment:
> [!IMPORTANT]
>  See inferred schema in ([build(DRAFT): Generate ISO datetime comparison](https://github.com/vega/vega-datasets/pull/645/commits/fb3ccc6dc2964dba7d6692369046664cfdaff392))

Switching to the actual [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format would resolve most of the *Incorrect schema detection* issues in https://github.com/vega/vega-datasets/issues/634#issuecomment-2510281715

https://github.com/vega/vega-datasets/blob/fb3ccc6dc2964dba7d6692369046664cfdaff392/datapackage.json#L923-L935

https://github.com/vega/vega-datasets/blob/fb3ccc6dc2964dba7d6692369046664cfdaff392/data/flights-1k.csv?plain=1#L1-L3

Reply:
### Update
As of (https://github.com/vega/vega-datasets/pull/645/commits/3418c5ada5e08357d628d120eb943c9d7bd012ae), I've replaced the previous `"iso"` alias with the full format string:

https://github.com/vega/vega-datasets/blob/0c11ec418acea2f8655d6b8bde166d5de16000b8/_data/flights.toml#L9-L14

This doesn't change the output, but I think is much more useful piece of metadata that can be reused elsewhere (e.g. https://datapackage.org/standard/table-schema/#datetime)

Reply:
@domoritz, @dsmedia 
Is there anything I can help clarify on this thread?

If you'd rather leave any column format string changes to another issue/PR - I can just remove `flights-1k.csv` and merge?

Reply:
Resolved in ([revert: remove `flights-1k.csv`](https://github.com/vega/vega-datasets/pull/645/commits/cd68193caaf78b7a80709a9dd5801c4fe5298ce8))